src/OSPSuite.UI/Views/Charts/CurveSettingsView.cs

```diff
@@ -389,10 +389,11 @@ private void onPopupMenuShowing(object sender, PopupMenuShowingEventArgs e)
          if (gridViewMenu == null)
             return;
 
-         if (gridView.GetSelectedRows().Length == 0)
+         //only one or no curve selected. We do not show the option
+         if (gridView.GetSelectedRows().Length <= 1)
```

Comment:
@georgeDaskalakis This was not working wiht only 1 selected. (e.g. the view was not showing up). Also I don't think it makes sense with only one selected item